Riot/Modules/Settings/Security/SecurityViewController.m

```diff
@@ -1618,13 +1664,14 @@ - (void)settingsKeyBackupTableViewSectionDidUpdate:(SettingsKeyBackupTableViewSe
 
 - (MXKTableViewCellWithTextView *)settingsKeyBackupTableViewSection:(SettingsKeyBackupTableViewSection *)settingsKeyBackupTableViewSection textCellForRow:(NSInteger)textCellForRow
 {
-    return [self textViewCellForTableView:self.tableView atIndexPath:[NSIndexPath indexPathForRow:textCellForRow inSection:SECTION_KEYBACKUP]];
+    NSIndexPath *indexPath = [self.tableViewSections exactIndexPathForRowTag:textCellForRow sectionTag:SECTION_KEYBACKUP];
+    return [self textViewCellForTableView:self.tableView atIndexPath:indexPath];
```

Comment:
`exactIndexPathForRowTag:sectionTag:` can return nil. I'm not sure but `dequeueReusableCellWithIdentifier:forIndexPath:` may have some problems with a `nil` indexPath.